editor/src/document/artboard_message_handler.rs

```diff
@@ -73,6 +73,20 @@ impl MessageHandler<ArtboardMessage, ()> for ArtboardMessageHandler {
 					);
 				}
 			}
+			ResizeArtboard { artboard, position, size } => {
+				responses.push_back(
+					ArtboardMessage::DispatchOperation(
+						DocumentOperation::SetLayerTransform {
+							path: artboard,
+							transform: DAffine2::from_scale_angle_translation(size.into(), 0., position.into()).to_cols_array(),
+						}
+						.into(),
```

Comment:
is this into even required here?

Reply:
@TrueDoctor It boxes the operation.

Reply:
I have converted it to a manual `Box::new` to add clarity

Reply:
Ah, right, always forget that